dnf-testing.sh

```diff
@@ -20,5 +20,8 @@ if [ ! "$PYTHON" ]; then
 fi
 
 "$PYTHON" container-test "$@"
+EXITCODE=$?
 
-popd || exit
+popd > /dev/null || :
```

Comment:
What is the `|| :` for? Would there be a more clear way to do it, this is just totally cryptic to anyone not knowing the exact semantics? From what I've quickly tried to find I would guess it actually doesn't do anything?